week-8/Homework/mandatory/4-programmer-humour/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
👍  Works well

The exercise did ask for a separate CSS file. Next time, you can try adding styling to it too